src/py/flwr/server/server.py

```diff
@@ -305,8 +305,10 @@ def disconnect_all_clients(self) -> None:
         """Send shutdown signal to all clients."""
         all_clients = self._client_manager.all()
         clients = [all_clients[k] for k in all_clients.keys()]
-        _ = shutdown(
-            clients=clients,
+        reconnect = Reconnect(seconds=None)
+        client_instructions = [(cp, reconnect) for cp in clients]
```

Comment:
```suggestion
        instruction = Reconnect(seconds=None)
        client_instructions = [(client, instruction) for client in clients]
```

Reply:
It shouldn't matter in the naming that they are `client_proxies`. If you think it does I would rather rename `clients` to `client_proxies`. Spent some time thinking whats going to be copied aka cp in bash here.